actionview/test/template/sanitize_helper_test.rb

```diff
@@ -40,4 +40,12 @@ def test_strip_tags_will_not_encode_special_characters
   def test_sanitize_is_marked_safe
     assert_predicate sanitize("<html><script></script></html>"), :html_safe?
   end
+
+  def test_sanitized_allowed_tags_class_method
+    assert(self.class.sanitized_allowed_tags)
```

Comment:
@rafaelfranca Do you mean like this? Or is it better to assert the content of return? Something like
`assert_includes(self.class.sanitized_allowed_tags, "strong")`

Reply:
I think you need to do something as follows: 

```
expected = ["a", "abbr", "acronym", "address", "b", "big", "blockquote", "br", "cite", "code", "dd", "del", "dfn", "dl", "dt", "em", "h1", "h2", "h3", "h4", "h5", "h6", "hr", "i", "img", "ins", "kbd", "li", "ol", "p", "pre", "samp", "small", "span", "strong", "sub", "sup", "tt", "ul", "var"]

assert_equal expected, ActionView::Base.sanitized_allowed_tags.sort
```

Similarly for `sanitized_allowed_attributes`
```
expected = ["abbr", "alt", "cite", "class", "datetime", "height", "href", "name", "src", "title", "width", "xml:lang"]

assert_equal expected, ActionView::Base.sanitized_allowed_attributes.sort
```

Reply:
added the exact expected set, thanks @abhaynikam ! however I'm not sure if we should involve `ActionView::Base` since this is a unit test for the helper module

Reply:
@waruboy In my opinion, your original proposal using `assert_includes` is a better approach.  The reason I originally suggested `assert_nothing_raised` was to avoid asserting on the exact values coming from [`rails-html-sanitizer`](https://github.com/rails/rails-html-sanitizer), since those may change independently of Rails.  For example, when a new tag is added to `rails-html-sanitizer`, the `assert_equal` will break.

An alternative would be to duplicate the logic entirely, e.g.:
```ruby
assert_equal Rails::Html::Sanitizer.safe_list_sanitizer.allowed_tags, ...
```
But, to me, that seems over-specified.

Reply:
@jonathanhefner that's what I prefer in my projects too, but I'm not too sure in this case. Shouldn't the Rails core be notified by the test if the values coming from `rails-html-sanitizer` changes, since that change Rails behaviour? Anyway, changed to use assert_includes

Reply:
I agree with @waruboy because changes at `rails-html-sanitizer` would hardly happen and even if it does I think the test should be broken to let everyone in the Rails team know about the changes and understand if any impact with the new addition. 

Also, `rails-html-sanitizer` version should be updated in order to test to start failing because of any change in the list and that PR should fix the expected value here as well.

IMO, we should go with `assert_equals` 😊 🤘